src/dc_receive_imf.rs

```diff
@@ -877,9 +892,14 @@ async fn add_parts(
                     part.param.to_string(),
                     part.bytes as isize,
                     is_hidden,
-                    mime_headers,
+                    if save_mime_headers || mime_modified {
+                        mime_headers.clone()
+                    } else {
+                        None
+                    },
```

Comment:
we already had a field that was used for exactly the purpose of saving the raw mime message, although the name `mime_headers` is a bit misleading - i meanwhile read it as "mime and headers" :)

Reply:
`save_mime_headers` as introduced at https://github.com/deltachat/deltachat-core/pull/388, not sure if it is actually in used, however, it is not much code, also, with this pr not even a wasted column anymore - so i would not remove it in this pr without further discussion.